### PR TITLE
RSDK-5742 return raw kinematics file from arm server

### PR DESCRIPTION
**Pull request by raybjork**
This PR makes it possible for an arm model within the RDK to return a kinematic representation in a format other than SVA, which is all that was previously possible.

### CODE REVIEW

File: components/arm/client_test.go [hunk 1]
```diff
@@ -63,7 +63,7 @@ func TestClient(t *testing.T) {
 		return errStopUnimplemented
 	}
 	injectArm.ModelFrameFunc = func() referenceframe.Model {
-		data := []byte("{\"links\": [{\"parent\": \"world\"}]}")
+		data := []byte("{\"links\": [{\"parent\": \"world\"t}]}")
```
Comment (nfranczak) [inline components/arm/client_test.go:66]: what is the `t` for?

**Review by nfranczak — APPROVED** [review]
looking solid! LGTM

File: components/arm/client_test.go (hunk 1 above)
Comment (raybjork) [reply components/arm/client_test.go:66]: oops

File: module/modmanager/manager_test.go [hunk 2]
```diff
@@ -803,8 +803,9 @@ func TestModuleMisc(t *testing.T) {
 		test.That(t, resp, test.ShouldNotBeNil)
 		modWorkingDirectory, ok := resp["path"].(string)
 		test.That(t, ok, test.ShouldBeTrue)
-		test.That(t, modWorkingDirectory, test.ShouldEqual, filepath.Dir(modPath))
-
+		// MacOS prepends "/private/" to the filepath so we check the end of the path to account for this
+		// i.e.  '/private/var/folders/p1/nl3sq7jn5nx8tfkdwpz2_g7r0000gn/T/TestModuleMisc1764175663/002'
+		test.That(t, modWorkingDirectory, test.ShouldEndWith, filepath.Dir(modPath))
```
Comment (zaporter-work) [inline module/modmanager/manager_test.go:808]: :+1:

**Pull request merged**